### PR TITLE
database upgrade

### PR DESCRIPTION
**Pull request by sniezekjp**

### CODE REVIEW

File: backend/models/Article.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  title: {
+    type: Sequelize.STRING
+  },
+  description: {
+    type: Sequelize.STRING
+  },
+  content: {
+    type: Sequelize.STRING
+  },
+  title: {
```
Comment (sniezekjp) [inline backend/models/Article.js]: two attributes with the the "title" key, remove one

File: backend/routes/articles.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
+const express = require('express');
+const router = express.Router();
+const db = require('../config/database');
+const Article = require('../models/Article');
+
+router.get('/', ( req, res ) => 
+	Article.findAll({ order: [['updatedAt', 'DESC']] })
+		.then(articles => {
+			console.log(articles);
+			res.send(articles);
```
Comment (sniezekjp) [inline backend/routes/articles.js]: I'd use `res.json` here instead of `res.send`.

File: backend/routes/articles.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      }
+    }).then( (result) => res.json(result) )
+  );
+
+  router.delete( "/:id", (req, res) =>
+    Article.destroy({
+      where: {
+        id: req.params.id
+      }
+    }).then( (result) => res.json(result) )
```
Comment (sniezekjp) [inline backend/routes/articles.js:46]: Let's use this format when we can: `.then(result => res.json(result));`

File: backend/models/Article.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  },
+  content: {
+    type: Sequelize.STRING
+  }
+});
+
+
+  Article.associate = function(models) {
+    // associations can be defined here
+  };
```
Comment (sniezekjp) [inline backend/models/Article.js:21]: Let's cleanup whitespaces here.

File: backend/routes/articles.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    Article.update({
+      title: req.body.title,
+      description: req.body.description,
+      content: req.body.content
+    },
+    {
+      where: {
+        id: req.params.id
+      }
+    }).then( (result) => res.json(result) )
```
Comment (sniezekjp) [inline backend/routes/articles.js]: Let's use this format when we can: .then(result => res.json(result));

File: backend/routes/articles.js [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+			res.send(articles);
+		})
+		.catch(err => console.log(err))
+	);
+
+router.get('/:id', ( req, res ) => 
+	Article.findByPk(req.params.id)
+		.then(article => res.json(article))
+		.catch(err => res.status(400).json('Error: ' + err))
+			);
```
Comment (sniezekjp) [inline backend/routes/articles.js]: Extra newline here

**Review by jsniezek05 — APPROVED** [review]
Looks good. Merging

**Pull request merged**